cloudman/values.yaml

```diff
@@ -148,14 +155,11 @@ cloudlaunch:
       hosts:
         - ~
       protocol: https
-keycloak:
-  persistence:
-    deployPostgres: false
 
+keycloak:
   keycloak:
     username: admin
     password: changeMe
```

Comment:
Also this. We should make this a required value I think, instead of a default.

Reply:
agreed